### ISSUE TITLE
Can't export OMPL file when there are subfolders

### ISSUE THREAD

**Issue by lollyP** [ancestorak/brief#313]
Thanks for all your hard work on the extension!

Exporting to an OMPL file seems to work fine where there is a simple folder structure (one level only), but when subfolders are added, clicking 'Export Feeds...' does nothing.

**Comment by tanriol** [comment]
Does it produce any messages in the debugging console? You can open the debugging tools with F12 on the Brief tab.

**Comment by lollyP** [comment]
That was quick! The good news is that when I tried to export to OPML, it worked. I wondered if that's because I'd restarted the browser in the interim, so I edited the folder structure to add an additional subfolder, then tried again to export to OPML (without a further browser restart), and it failed. In terms of debugging, I'm not sure if this is what you want, but the console records:

incorrect database opml.js:71:21

exportFeeds moz-extension://[MY-CODE-NUMBER]/scripts/opml.js:71:21
dropdown-export moz-extension://[MY-CODE-NUMBER]/ui/feedlist.js:999:38

Nb: refreshing the tab or closing and reopening the extension tab does not impact the ability to export (ie, when the folder has been modified such that it wont export to OPML, closing and reopening the extension doesn't resolve the problem).
